.travis.yml

```
@@ -1,32 +1,21 @@
+# Enviourment variables
 language: c
 cache:
   ccache: true
   pip: true
-os:
-  - linux
-  - osx
 dist: bionic
 osx_image: xcode11
-compiler:
-  - clang
-  - gcc
+env: build_type=release
 addons:
   apt:
-    packages:
-      - cmake
-      - yasm
-      - valgrind
-      - libgstreamer-plugins-base1.0-dev
-      - libgstreamer1.0-dev
+    sources:
+    packages: cmake yasm valgrind libgstreamer-plugins-base1.0-dev libgstreamer1.0-dev
   homebrew:
-    packages:
-      - yasm
-      - ccache
+    packages: yasm ccache
```

```suggestion
    packages:
      - yasm
      - ccache
```
Travis homebrew doesn't handle single like packages for macOS ( tries to find a package called "yasm ccache")